src/project/internal/projectactionscontroller.cpp

```diff
@@ -1577,6 +1581,28 @@ void ProjectActionsController::warnScoreCouldnotBeSaved(const std::string& error
     interactive()->warning(muse::trc("project/save", "Your score could not be saved"), errorText);
 }
 
+void ProjectActionsController::warnScoreCorruptAfterSave(const Ret& ret)
+{
+    std::string errMessage = ret.toString();
+
+    std::any corruptedAfterMove = ret.data("corruptedAfterMove");
+    std::string originalFileCorruptedMessage = corruptedAfterMove.has_value() && std::any_cast<bool>(corruptedAfterMove)
+                                               ? muse::trc("project/save", "Sadly, your file has become corrupt.\n\n")
```

Comment:
Seems the Linux builds complains about the trailing whitespace, most probably the 2nd `\n`

Reply:
Beter append those trailing new lines untranslatable

Reply:
`? muse::trc("project/save", "Sadly, your file has become corrupt.") + "\n\n"`